Files/Views/LayoutModes/GridViewBrowser.xaml

```diff
@@ -15,7 +15,7 @@
     xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
     x:Name="PageRoot"
     Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
-    NavigationCacheMode="Disabled"
+    NavigationCacheMode="Enabled"
```

Comment:
Is this change intentional?

Reply:
I didn't touch any XAML file, I think there's something wrong with rebase.

Reply:
Fixed in c63a9b7